### PR TITLE
chore(readme): add logo  + link of foundation

### PR DESCRIPTION
**Pull request by AugustinMauroy**
Update remade to have same flavor of CPC repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
-# OpenJS Foundation Standards Working Group
+<a href="https://openjsf.org">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color-dark_background.svg" />
+    <img src="https://raw.githubusercontent.com/openjs-foundation/artwork/main/openjs_foundation/openjs_foundation-logo-horizontal-color.svg" width="200" />
```
Comment (ljharb) [inline README.md:4]: needs an alt

Comment (AugustinMauroy) [reply README.md]: Thanks for you review !
Done !

**Review by ljharb — APPROVED** [review]

**Review by DerekNonGeneric — APPROVED** [review]
The [rich diff](https://github.com/openjs-foundation/standards/pull/229/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) looks good!

**Comment by jorydotcom** [comment]
love it! thanks @AugustinMauroy !!

**Pull request merged**